src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/policies/PolicyV1.kt

```diff
@@ -148,18 +156,18 @@ val POLICY_V1_0 = LicencePolicy(
         inputs = listOf(
           Input(
             addAnother = AddAnother("Add another person"),
-            case = "capitalise",
+            case = CAPITALISED,
```

Comment:
Behaviour will change here - `capitalise` wasn't a valid case type so would silently ignore

Now this will correctly title case the Name.